quarkc/test/compile/bad-method-expr.err

```diff
@@ -0,0 +1 @@
+bad-method-expr.q:4:24:type mismatch: expected 1.quark.String, got 0.bad_method_expr.Foo.bar
```

Comment:
On the one hand that's a legit error. ON the other hand... if we don't support references to functions or methods, shouldn't it fail do to that? Also where did `Foo` come from?